docs/status.md

```diff
@@ -71,7 +71,7 @@ one of the following tracking labels.
 | convolution              | no            | yes*         | yes*           | revisit         | no          |
 | cosine                   | yes           | yes          | yes            | yes             | yes         |
 | count_leading_zeros      | yes           | yes          | yes            | yes             | no          |
-| create_token             | no            | revisit      | no             | yes             | no          |
+| create_token             | no            | revisit      | yes            | yes             | no          |
```

Comment:
Both "Verification" and "Type Inference" should be "yes*", because we don't yet have a spec for this op.